src/GraphQL/Types/Scalars/DateTimeOffsetGraphType.cs

```diff
@@ -34,6 +34,7 @@ public DateTimeOffsetGraphType()
         public override object? ParseValue(object? value) => value switch
         {
             DateTimeOffset _ => value,
+            DateTime d => d.Kind == DateTimeKind.Unspecified ? new DateTimeOffset(d, TimeSpan.Zero) : new DateTimeOffset(d),
```

Comment:
So... that is you want to handle `Unspecified` as `UTC`, right?

Reply:
Exactly, at least to preserve the same behavior as with `.AddSystemTextJson()`

Reply:
OK